Raw json:

{'hasError': False, 'errParameter': [], 'reason': []}

{'hasError': False, 'errParameter': [], 'reason': []}

{'hasError': True, 'errParameter': ['alluxio.underfs.web.connnection.timeout=80hhhhh', 'alluxio.network.host.resolution.timeout=1sec', 'alluxio.security.stale.channel.purge.interval=6day'], 'reason': ["Invalid value for alluxio.underfs.web.connnection.timeout. It should be a number followed by 'ms', 's', 'm', 'h', or 'd' (milliseconds, seconds, minutes, hours, or days).", "Invalid value for alluxio.network.host.resolution.timeout. It should be a number followed by 'ms', 's', 'm', 'h', or 'd' (milliseconds, seconds, minutes, hours, or days).", "Invalid value for alluxio.security.stale.channel.purge.interval. It should be a number followed by 'ms', 's', 'm', 'h', or 'd' (milliseconds, seconds, minutes, hours, or days)."]}

Final result:

The CONFIGURATION FILE IS CORRECT
